resources.whatwg.org/build/deploy.sh

```diff
@@ -64,36 +64,42 @@ header() {
   echo -e "\x1b[1m$1\x1b[0m"
 }
 
+curlretry() {
+  curl --fail --retry 2 "$@"
+}
+
+bikeshed() {
```

Comment:
Maybe for clarity "curlbikeshed" or "remotebikeshed"? Otherwise it's too easy to be reading things and thinking we've somehow locally installed bikeshed.